src/app/utils/push.ts

```diff
@@ -0,0 +1,7 @@
+export function getStatuses(data: string): string {
```

Comment:
I'm not sure why you are jumping forth and back with object and JSON string. This can be made very simple. Just use `JSON.stringify` when needed.